src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import useFetch from 'utils/hooks/useFetch';
+import { IPatient } from 'types/IPatient';
+
+const PatientInfo = ({ patientID }: any) => {
+    const { response } = useFetch<IPatient>('fhir/Patient/' + patientID);
+
+    console.log(response);
```

Comment:
I'm not a big fan of console logs in production code. What do you think?

Reply:
I agree